pkgs/tools/networking/unbound/default.nix

```diff
@@ -59,6 +59,13 @@ stdenv.mkDerivation rec {
     "--with-libnghttp2=${libnghttp2.dev}"
   ];
 
+  # Remove references to compile-time dependencies that are included in the configure flags
+  postConfigure = let
+    inherit (builtins) storeDir;
+  in ''
+    sed -E '/CONFCMDLINE/ s;${storeDir}/[a-z0-9]{32}-;${storeDir}/eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee-;g' -i config.h
```

Comment:
I should probably just use `nuke-refs` from the `nukeReferences` package here even though that has potential to backfire if there are other paths in there that must exist during runtime.